.github/workflows/build.yml

```diff
@@ -21,6 +22,33 @@ jobs:
       matrix:
         java: ['17', '21']
     steps:
+      - name: Checkout jkube-images repo
+        uses: actions/checkout@v4
+        with:
+          repository: eclipse-jkube/jkube-images
+          ref: v0.0.24
+      - name: Install CEKit
+        uses: cekit/actions-setup-cekit@v1.1.6
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Build multi-arch jkube-java image
+        if: github.event_name == 'push'
+        env:
+          USERNAME: ${{ secrets.QUAY_USERNAME }}
+          PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+        run: |
+          docker login --help
+          docker login -u $USERNAME -p $PASSWORD quay.io
+
+          if [[ "${{ matrix.java }}" == "17" ]]; then
+            cekit --descriptor jkube-java-17.yaml build --dry-run docker
+          else
+            cekit --descriptor jkube-java.yaml build --dry-run docker
+          fi
+          docker buildx build -t quay.io/hawtio/jkube-java-${{ matrix.java }}:latest \
+            --platform linux/amd64,linux/arm64,linux/ppc64le,linux/s390x \
+            --provenance=false \
+            --push target/image
```

Comment:
This will need to be pushed to quay right? Still not very happy with having to build our own images for building - it should be provided for SB when it's officially supported.